Makefile

```diff
@@ -1,11 +1,12 @@
-PREFIX=/usr/local
-BINDIR=$(DESTDIR)$(PREFIX)/bin
-MANDIR=$(DESTDIR)$(PREFIX)/share/man/man1
-DOCDIR=$(DESTDIR)$(PREFIX)/share/doc/googler
-UNAME_S:=$(shell uname -s)
+PREFIX ?= /usr/local
+BINDIR = $(DESTDIR)$(PREFIX)/bin
+MANDIR = $(DESTDIR)$(PREFIX)/share/man/man1
+DOCDIR = $(DESTDIR)$(PREFIX)/share/doc/googler
+BASHCOMPDIR = $(DESTDIR)$(PREFIX)/etc/bash_completion.d
+FISHCOMPDIR = $(DESTDIR)$(PREFIX)/share/fish/vendor_completions.d
+ZSHCOMPDIR = $(DESTDIR)$(PREFIX)/share/zsh/site-functions
```

Comment:
We are effective installing the scripts to:
```
/usr/local/etc/bash_completion.d
/usr/local/share/fish/vendor_completions.d
/usr/local/share/zsh/site-functions
```

instead of the intended

```
/etc/bash_completion.d
/usr/share/fish/vendor_completions.d
/usr/share/zsh/site-functions